algosdk/util.py

```diff
@@ -28,7 +29,7 @@ def algos_to_microalgos(algos):
     Returns:
         int: how many microalgos
     """
-    return algos*constants.microalgos_to_algos_ratio
+    return round(algos*constants.microalgos_to_algos_ratio)
```

Comment:
We `round` here because we only want whole-number microAlgos?

Reply:
Yes; since microAlgos never get split, I don't think we should have fractional microAlgo amounts.